spec/intro.dd

```diff
@@ -8,59 +8,51 @@ $(SPEC_S Introduction,
         native executable.
         )
 
-$(H2 Phases of Compilation)
+$(H2 Conceptual models)
 
-$(P The process of compiling is divided into multiple phases. Each phase has no
-dependence on subsequent phases. For example, the scanner is not perturbed by
-the semantic analyzer. This separation of the passes makes language tools like
-syntax directed editors relatively easy to produce. It also is possible to
-compress D source by storing it in $(SINGLEQUOTE tokenized) form.)
+$(H3 Environments)
 
-$(OL
-        $(LI $(B source character set)$(BR)
```

Comment:
Please keep this section, it probably doesn't belong in Compilation phases though

Reply:
The reason I removed this point and the next is that they appear to dive into some details of lexical analysis. The lexical analysis process is described fully in a following section so picking a couple of the details and mentioning them here seems odd.

Reply:
Hmm, OK.